tests/unit/preprocessor/_time/test_time.py

```diff
@@ -149,6 +156,51 @@ def test_extract_time_no_time(self):
         assert cube == sliced
 
 
+class TestExtractTimerange(tests.Test):
```

Comment:
```suggestion
class TestClipStartEndYear(tests.Test):
```
Should we rename this one as well?